docs/installation.md

```diff
@@ -15,6 +16,9 @@ First, add our official ralph repository:
     sudo apt-key adv --keyserver  hkp://keyserver.ubuntu.com:80 --recv-keys 379CE192D401AB61
     sudo sh -c "echo 'deb https://dl.bintray.com/vi4m/ralph wheezy main' >  /etc/apt/sources.list.d/vi4m_ralph.list"
 
+After, install the secure transport for accessing repository :
```

Comment:
@cdb-fr I mean to move info about secure transport installation before adding new ralph's apt-key - it'd be more straightforward (and will not crash if someone will execute `sudo apt-get update` in a meanwhile)